app/src/main/java/com/example/atheneum/models/GoodreadsReviewInfo.java

```diff
@@ -0,0 +1,81 @@
+package com.example.atheneum.models;
+
+/**
+ * Model class for holding the goodreads review info.
```

Comment:
You should add unit tests for this since we need unit tests for all model classes.

Reply:
Done